<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiInvite</class>
 <widget class="QDialog" name="uiInvite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>92</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>472</width>
    <height>92</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>472</width>
    <height>92</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Invitation</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../scc.qrc">
    <normaloff>:/images/logo16x16.png</normaloff>:/images/logo16x16.png</iconset>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_msg">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Invite</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_accept">
        <property name="text">
         <string>Accept</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_reject">
        <property name="text">
         <string>Reject</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_ignore">
        <property name="text">
         <string>Ignore</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_options">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Options</string>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Close</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../scc.qrc"/>
 </resources>
 <connections/>
</ui>
